.circleci/config.yml

```diff
@@ -1,7 +1,31 @@
 version: 2.1
 
-
 jobs:
+  build-geth:
+    docker:
+      - image: cimg/go:1.18
+    resource_class: medium
```

Comment:
how do we choose this value?

Reply:
https://circleci.com/docs/2.0/configuration-reference/#resourceclass
It's cheaper & doesn't seem to slow tests down that much (we don't run individual unit tests in parallel))